.travis.yml

```diff
@@ -52,6 +52,16 @@ install:
      ./autogen.sh --prefix=/usr
      make && sudo make install)
 
+  # Install xcb-errors if needed
+  - |
+    set -e
+    if [[ "$WITH_XCB_ERRORS" == "yes" ]]; then
+      git clone --depth 1 --recursive https://gitlab.freedesktop.org/xorg/lib/libxcb-errors.git /tmp/xcb-errors
+      (cd /tmp/xcb-errors
+       ./autogen.sh --prefix=/usr
+       make && sudo make install)
```

Comment:
Also needs xcb-proto it seems:
```
checking for XCBPROTO... no
configure: error: Package requirements (xcb-proto) were not met:
No package 'xcb-proto' found
```

Reply:
Maybe APT addons can be used?  https://docs.travis-ci.com/user/installing-dependencies/  (this can be configured for single jobs)

Reply:
Heh, seems like we were looking at this simultaneously. :-)
I just added xcb-proto to our big list of packages to install.

Reply:
I would prefer to have it only with / nearby where it is needed.

Reply:
Sorry, what do you mean? I should move the installation of `xcb-proto` to a separate invocation of `apt-get install`?

Reply:
Yes.

Reply:
Then it is easier in the future to see what is required because of what (and it is a bit less overhead , and more explicit).

Reply:
Yup, you are right.